bobtemplates/plone_addon/.travis.yml.bob

```diff
@@ -1,28 +1,26 @@
 language: python
+python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
```

Comment:
I changing this only for the packages created with bobtemplates.plone, not for bobtemplates.plone itself; the `eggs` directory I'm caching is the one create by buildout.

why you want to keep that on the other packages? what problem are you solving?